caim_base/views/awg/applications.py

```diff
@@ -76,26 +78,38 @@ def update_application(request, awg_id, application_id):
     application.reject_reason_detail = request.POST.get("reject_reason_detail")
     application.save()
 
+    notify = {
+        FosterApplication.Statuses.ACCEPTED: notify_caim_foster_application_accepted,
+        FosterApplication.Statuses.REJECTED: notify_caim_foster_application_rejected,
+    }
+    notify[application.status](application)
+    
+    templates = {
+        FosterApplication.Statuses.ACCEPTED: "awg/manage/applications/accept_confirmed_modal.html",
+        FosterApplication.Statuses.REJECTED: "awg/manage/applications/reject_confirmed_modal.html",
+    }
+    template = templates[application.status]
+
     context = {
         **context,
         "awg": awg,
-        "applications": query_applications_for_awg(awg),
-        "application_status_options": [c[0] for c in FosterApplication.Statuses.choices],
+        "app": application,
+        # "applications": query_applications_for_awg(awg),
+        # "application_status_options": [c[0] for c in FosterApplication.Statuses.choices],
```

Comment:
if we dont need these lines, should just remove them rather than commenting them out